lib/workers/repository/config-migration/branch/rebase.ts

```diff
@@ -24,10 +20,7 @@ export async function rebaseMigrationBranch(
   const configFileName = migratedConfigData.filename;
   const contents = migratedConfigData.content;
   const existingContents = await getFile(configFileName, branchName);
-  if (
-    contents === existingContents &&
-    !(await isBranchBehindBase(branchName))
-  ) {
```

Comment:
`isBranchBehindBase` will always return `true` when a commit was added to the base branch even when config file contents are unchanged, causing unnecessary rebasing.

Reply:
@rarkins not sure which behavior we prefer here.

Reply:
`contents` is what to be committed, `existingContents` is the current content in the file we want to commit to, i dont think `isBranchBehindBase` adds much, if any it will force unneeded rebases